tests/debugger/oegdb/enc/enc.c

```diff
@@ -44,6 +60,9 @@ int square(int x)
         oe_is_within_enclave(&x, sizeof(x)) ? "within enclave"
                                             : "outside enclave");
     g_square_called = true;
+
+    // Raising signal in a function called by gdb crashes gdb 8.0 and above.
```

Comment:
For my understanding, is this expected?

Reply:
gdb 8.x seems like a downgrade from gdb 7.x. Lots of stability issue. It is the default in Ubuntu 19.10 which I'm running.

Reply:
From previous run on RHEL 8.
https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/rest/organizations/jenkins/pipelines/Bors/branches/trying/runs/209/nodes/66/steps/453/log/?start=0

```bash
Program received signal SIGILL, Illegal instruction.
0x00007ffff50034f6 in _raise_cpuid_exception () at ../tests/debugger/oegdb/enc/enc.c:39
39	    asm("movl  $1, %%eax   \n\t"
oegdb: Ignoring emulated instruction cpuid.
Continuing.
square called with 100
x lies within enclave
../../gdb/infcall.c:1188: internal-error: value* call_function_by_hand_dummy(value*, type*, int, value**, void (*)(void*, int), void*): Assertion `call_thread->thread_fsm == &sm->thread_fsm' failed.
A problem internal to GDB has been detected,
further debugging may prove unreliable.
Quit this debugging session? (y or n) [answered Y; input not from terminal]
```